src/ClusterManager/virtual_cluster_status.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python3
+
+import collections
+import json
+import logging
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(
+    os.path.abspath(__file__)), "../utils"))
+
+from resource_stat import Cpu, Memory, Gpu
+from cluster_resource import ClusterResource
+from cluster_status import ClusterStatus, get_jobs_without_pods
+from job_params_util import get_resource_params_from_job_params
+from quota import calculate_vc_resources
+
+logger = logging.getLogger(__name__)
+
+
+def get_vc_info(vc_list):
+    vc_info = {}
+    for vc in vc_list:
+        resource_quota = {}
+        try:
+            resource_quota = json.loads(vc["resourceQuota"])
+        except:
+            logger.exception("Parsing resourceQuota failed for %s", vc,
+                             exc_info=True)
```

Comment:
I think exception do not need exec_info=True argument, it's by default in exception method